service/src/main.rs

```diff
@@ -23,11 +23,41 @@ use serde::{Deserialize, Serialize};
 use sled::{Transactional, Tree as SledTree};
 
 use base64::Engine;
+use hex;
+use sha3::{Digest, Sha3_256};
 
-const KECCAK_INCLUSION_ELF: &[u8] = include_bytes!(
+/// A Succunct Prover Network request ID.
+/// See: https://docs.succinct.xyz/docs/generating-proofs/prover-network/usage
+type SuccNetJobId = [u8; 32];
+
+/// A SHA3 256 bit hash of a zkVM program's ELF.
+type SuccNetProgramId = [u8; 32];
```

Comment:
Silly q maybe: https://docs.succinct.xyz/docs/writing-programs/compiling#production-builds/ seems to imply there is no canonical way to compute a progam ID of some kind, that is a first class item in R0 land https://dev.risczero.com/terminology#image-id

is there no sp1 provided program ID?